microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,191 @@
+## Layers in 3D View
+
+<!-- Title: Layers in 3D View -->
+<!-- Subtitle: The Layers tool isn't going away - find it in the 3D View tool today. -->
+
+Since [Microsoft Edge 88](../../2020/11/devtools.md#composited-layers-are-now-in-3d-view), the **3D View** tool has provided an improved version of the **Layers** tool.  Now, in Microsoft Edge 99, the standalone **Layers** tool has been removed, and contains a link that redirects you to the **Composited Layers** view within the **3D View** tool.  You can find all the functionality from the **Layers** tool in the **3D View** tool.
+
+![The Layers tool now links to the 3D View tool with the Composited Layers view open.](../../media/2022/03/layers-3d-view-tool.png)
+<!-- work item > layersmove.gif -->
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../../../3d-view/index.md).
+
+
+<!-- ====================================================================== -->
+## Use your source maps to unminify performance profiles
+
+<!-- Title: The Performance tool can now display unminified call stacks -->
+<!-- Subtitle: Use the new Unminify button in the Performance tool to download an unminified version of the performance profile you recorded. -->
+
+In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack was not using source maps, even if you hosted them on your server, to unminify the call stack.
+
+Starting in Microsoft Edge version 99, you can unminify a call stack in a performance profile, as follows:
+1. After recording a profile, in the **Performance** tool, click the new **Unminify** button.
+1. Download the unminified performance profile.<!--how?-->
+1. Import the unminified performance profile into the **Performance** tool.<!--how?-->
+
+The **Unminify** button will use source maps, provided that they are hosted side-by-side with your production code, to unminify the call stack in the **Performance** tool.
+
+The minified call stack in the **Performance** tool shows function names as **b**, **O**, **Xt**, and **bn**:
+
+![The minified call stack in the Performance tool.](../../media/2022/03/minified-call-stack-performance-tool.png)
+<!-- Image 1: Minified call stack in Performance tool
+Navigate to: https://outlook-sdf.office.com/mail/
+Open DevTools > Performance
+Record a performance profile while scrolling through your inbox
+Select a small section of the profile and look at the call stack       1380ms-1400ms
+Draw red highlights around function names like n, O, i, Xt, Bn, Yn. It won't be these exactly in your screenshot but you can see these are minified function names
+Remember the section of the profile that you are looking at
+-->
+
+The unminified call stack in the **Performance** tool now shows those functions labeled with meaningful names: **invokeFunc (b)**, **executeAction (O)**, **endBatch (Xt)**, and **runReactions (bn)**:
```

Comment:
@zoherghadyali @captainbrosset @Reezaali - change "call stack" to "flame chart" in this What's New item?  Like per captainbrosset's comments on the Unminify PR.

Reply:
Yes, I agree with that. Flame chart is the correct term